openmc/mgxs/mgxs.py

```diff
@@ -2452,7 +2866,7 @@ def __init__(self, domain=None, domain_type=None,
         self._rxn_type = 'nu-scatter'
 
 
-class ScatterMatrixXS(MGXS):
+class ScatterMatrixXS(MatrixMGXS):
```

Comment:
Ok, I forgot about the moments in my red eye flight stupor. Sounds good, keep those methods as they are, but do your own set of checks to ensure that each of those methods work for each of the `MarrixsMGXS` subclasses (esp. `NuFissionMatrixXS`).